packages/apps/reputation-oracle/server/src/modules/escrow-completion-tracking/escrow-completion-tracking.service.ts

```diff
@@ -70,4 +93,177 @@ export class EscrowCompletionTrackingService {
+            );
+          }
+
+          await this.payoutService.executePayouts(
+            escrowCompletionTrackingEntity.chainId,
+            escrowCompletionTrackingEntity.escrowAddress,
+            escrowCompletionTrackingEntity.finalResultsUrl,
+            escrowCompletionTrackingEntity.finalResultsHash,
+          );
+        }
+
+        escrowCompletionTrackingEntity.status =
+          EscrowCompletionTrackingStatus.PAID;
+        await this.escrowCompletionTrackingRepository.updateOne(
+          escrowCompletionTrackingEntity,
+        );
+      } catch (err) {
+        const errorId = uuidv4();
+        const failureDetail = `${ErrorEscrowCompletionTracking.PendingProcessingFailed} (Error ID: ${errorId})`;
+        this.logger.error(
+          `Error processing escrow completion tracking. Error ID: ${errorId}, Escrow completion tracking ID: ${escrowCompletionTrackingEntity.id}, Reason: ${failureDetail}, Message: ${err.message}`,
+        );
+        await this.handleEscrowCompletionTrackingError(
+          escrowCompletionTrackingEntity,
+          failureDetail,
+        );
+        continue;
+      }
+    }
+  }
+
+  public async processPaidEscrowCompletion(): Promise<void> {
+    const escrowCompletionTrackingEntities =
+      await this.escrowCompletionTrackingRepository.findByStatus(
+        EscrowCompletionTrackingStatus.PAID,
+      );
+
+    // TODO: Add DB transactions
+    for (const escrowCompletionTrackingEntity of escrowCompletionTrackingEntities) {
+      try {
+        const { chainId, escrowAddress } = escrowCompletionTrackingEntity;
+
+        const signer = this.web3Service.getSigner(chainId);
+        const escrowClient = await EscrowClient.build(signer);
+
+        const escrowStatus = await escrowClient.getStatus(escrowAddress);
+        if (escrowStatus === EscrowStatus.Paid) {
+          await escrowClient.complete(escrowAddress, {
+            gasPrice: await this.web3Service.calculateGasPrice(chainId),
+          });
+
+          // TODO: Technically it's possible that the escrow completion could occur before the reputation scores are assessed,
+          // and the app might go down during this window. Currently, there isn’t a clear approach to handle this situation.
+          // Consider revisiting this section to explore potential solutions to improve resilience in such scenarios.
+          await this.reputationService.assessReputationScores(
+            chainId,
+            escrowAddress,
+          );
+        }
+
+        const callbackUrls = [
+          (
+            await OperatorUtils.getLeader(
+              chainId,
+              await escrowClient.getJobLauncherAddress(escrowAddress),
+            )
+          ).webhookUrl,
+          (
+            await OperatorUtils.getLeader(
+              chainId,
+              await escrowClient.getExchangeOracleAddress(escrowAddress),
+            )
+          ).webhookUrl,
+          (
+            await OperatorUtils.getLeader(
```

Comment:
It was disabled in `cron`, was it uncommented intentionally?

Reply:
Yes, about 2 weeks ago @Dzeranov asked me to uncomment it. @Dzeranov is it correct?

Reply:
True. 
But I think you can comment sending webhook to the recording oracle back. I checked CVAT oracles src and found out that they're expecting this webhook only on Exchange Oracle.

Reply:
Okay, thanks